templates/inscription.html

```diff
@@ -19,6 +19,17 @@ <h1>Inscription {{ self.number }} (unstable)</h1>
 <dl>
   <dt>id</dt>
   <dd class=monospace>{{ self.inscription_id }}</dd>
+%% if !self.inscription.metadata().is_empty() {
+  <dt>metadata</dt>
+  <dd>
+    <dl>
+%% for (key, value) in self.inscription.metadata() {
+      <dt>{{ key }}</dt>
+      <dd>{{ value }}</dd>
+%% }
+    </dl>
+  </dd>
+%% }
 %% if let Some(parent) = &self.parent {
```

Comment:
Do you think we should  also add these to the metatags on the head of the `page.html` if they key to open graph keys? That way someone could have the title of the inscription page be about what the inscription is instead of the inscription number if they desire as well as a description. Then links to this page could show more about the inscription on say twitter. Or would that not be ideal because then people can shit post there haha.
See comment: 
https://github.com/ordinals/ord/pull/2421#issuecomment-1705677285

Reply:
I think we wouldn't want to add arbitrary metadata, although if there is a `title` and `description` field present, those could be added. This is probably out-of-scope for the initial spec and PR though, just because it's ideal to keep PRs as small as possible so they stay managable.

Reply:
Yeah for sure agreed.